### PR TITLE
Allows you to toggle off Byond/Discord/URLs in OOC

### PR DESCRIPTION
**Pull request by GeneriedJenelle**
Just toggleable OOC options that you can enable or disable at any time. Might be useful if you guys get hit.

### CODE REVIEW

File: config/example/config.txt [hunk 1]
```diff
@@ -518,4 +518,10 @@ SQLITE_FEEDBACK_MIN_AGE 7
...
 ## Comment this out if you don't want to use the 'nightshift lighting' subsystem to adjust lights based on ingame time
-ENABLE_NIGHT_SHIFTS
\ No newline at end of file
+ENABLE_NIGHT_SHIFTS
+## OOC/LOOC control ##
+
+# Uncomment to allow links of the following kinds. #
+# ALLOW_BYOND_LINKS
+# ALLOW_DISCORD_LINKS
+ALLOW_URL_LINKS
```
Comment (TheFurryFeline) [inline config/example/config.txt:527]: Didn't you make a comment about having ALLOW_URL_LINKS disabled by default? This example/config.txt enables it by default if I'm not wrong.

Comment (GeneriedJenelle) [reply config/example/config.txt:527]: It's the other way around. If ALLOW_URL_LINKS = 1 (this config here makes it 1), it allows URLs to be enabled. See here:

> if((findtext(msg, "http://") || findtext(msg, "https://")) && !config.allow_url_links)

Comment (TheFurryFeline) [reply config/example/config.txt:527]: Ah, the OR operator. Fair enough, my mistake.

**Comment by BDpuffy420** [comment]
Def need this for Halo:SSE

**Pull request merged**